usr/src/boot/sys/boot/forth/menu-commands.4th

```diff
@@ -138,6 +138,26 @@ create chaincmd 1030 chars allot
 	0 boot ( state -- )
 ;
 
+\
+\ Platform-image selection for standalone SmartOS is mostly in pi.rc.
+\ We steal a boot environment routine too in pi_draw_screen (way below).
```

Comment:
this comment feels a little clumsy. I'm not sure what it's saying. Maybe:

    \ We will also steal a boot environment routine later in pi_draw_screen (way below).

or

    \ We also steal a boot environment routine here that getsused in pi_draw_screen (way below).

Reply:
Using:
`\ We will also steal a boot environment routine in pi_draw_screen (way below).`